Add docker-compose configuration

hdigital opened the pull request:
@miherrmann please review this version with _docker-compose_ to launch the container.

I find it more accessible with fewer options to launch the container.

It also avoids the previous issue with different configurations for most OSs and Windows.

hdigital committed:
Add docker-compose configuration

miherrmann commented:
To run shell inside container created with 'docker-compose' change container name to:

docker exec -it partypositions-wikitags_rocker_1 bash

hdigital committed:
Improve Docker documentation
+ add info about Docker details section below
+ add link to Docker Debian install documentation
+ add information about Docker resources settings

hdigital committed:
Remove documentation for Docker only usage

hdigital commented:
@miherrmann I agree the the Docker only documentation is not needed.

I removed it and merged the branch into main.
